packages/node_modules/@ciscospark/redux-module-conversation/src/reducer.js

```diff
@@ -65,17 +58,18 @@ export default function reducer(state = initialState, action) {
     let activities = new OrderedMap(conversationActivities.map((activity) => [activity.url, activity]));
     activities = activities.sortBy((activity) => activity.published);
 
-    return Object.assign({}, state, {
-      activities,
-      defaultActivityEncryptionKeyUrl,
-      id,
-      kmsResourceObjectUrl,
-      locusUrl,
-      url,
-      isFetching: false,
-      isLoaded: true,
-      participants: participants.items
-    });
+    return state
+      .set(`activities`, activities)
+      .merge({
+        defaultActivityEncryptionKeyUrl,
```

Comment:
Actually i'm wondering if we should keep everything as an immutable object until it hits an sdk method and then we convert with `.toJS()`. That may actually speed stuff up.